src/components/book.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] [Optional] Let's not disable linter rules. These are here to help us write a clean block of code.